README.md

```diff
@@ -7,3 +7,29 @@ permutation groups of degree < 4096.
 
 It has been formerly a part of the core GAP system, and has been
 converted to a separate GAP package in 2017 for the GAP 4.9 release.
+
+## Documentation
+
+Full information and documentation can be found in the manual, available
+as PDF `doc/manual.pdf` or as HTML `doc/chap0.htm`, or on the package
+homepage at
+
+  <http://gap-packages.github.io/primgrp/>
+
+
+## Bug reports and feature requests
+
+Please submit bug reports and feature requests via our GitHub issue tracker:
+
+  <https://github.com/gap-packages/primgrp/issues>
+
+
+# License
+
+PrimGrp is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by the
+Free Software Foundation; either version 2 of the License, or (at your
+option) any later version.
+
+For details see the files COPYRIGHT.md and LICENSE.
```

Comment:
One more thing - there is no COPYRIGHT.md file in the package. Only the LICENSE file.